charts/sourcegraph-appliance/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   image: appliance
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "{{ .Chart.AppVersion }}"
+  tag: &version "5.5.3738"
```

Comment:
In the long run, https://linear.app/sourcegraph/issue/REL-322/appliance-helm-chart-should-be-versioned should replace this at release time.